perfkitbenchmarker/pkb.py

```diff
@@ -143,8 +141,16 @@
     'scratch_disk_type', None,
     [disk.STANDARD, disk.REMOTE_SSD, disk.PIOPS, disk.LOCAL],
     'Type for all scratch disks. The default is standard')
+flags.DEFINE_string(
+    'data_disk_type', None,
+    'Type for all benchmark disks.')
```

Comment:
I added the info in the help text here. I think a comment would make the most sense in disk.py, since that has information on disks. I can add one once I upload to the wiki.